src/main/java/org/jenkinsci/plugins/pipeline/utility/steps/AbstractFileCompressStep.java

```diff
@@ -114,4 +115,25 @@ public void setOverwrite(boolean overwrite) {
         this.overwrite = overwrite;
     }
 
+    /**
+     * <a href="https://ant.apache.org/manual/dirtasks.html#defaultexcludes" target="_blank">Ant style pattern</a>
+     * of files to disable default excludes from the archive.
+     *
+     * @return the defaultExcludes boolean
+     */
+    public boolean isDefaultExcludes() {
```

Comment:
Because this feature only applies to Zip files, and the configuration options is only in that class, this code should be moved into the `ZipStep` class.

Reply:
Moved it to ZipStep class. Can you review please.